Store.Db/Repositories/ClientRepository.cs

```diff
@@ -1,43 +1,61 @@
 ﻿using Store.Db.Interfaces;
 using Store.Db.Entities;
 using Store.Db.Models;
+using RockLib.Logging;
 
 namespace Store.Db.Repositories
 {
     public class ClientRepository : IClientRepository
     {
         private readonly StoreDbContext _context;
-
-        public ClientRepository(StoreDbContext context) 
+        private readonly ILogger _logger;
+        public ClientRepository(StoreDbContext context, ILogger logger) 
         {
-            this._context = context;
+            _context = context;
+            _logger = logger;
         }
 
         public ClientsRecordView GetAllOrders(int id)
         {
-            var clientRecords = new ClientsRecordView
+            try
             {
-                ClientName = (from c in _context.Customers
-                              where c.CustomerId == id
-                              select new ClientsRecord
-                              {
-                                  Name = c.FirstName + " " + c.LastName,
-                              }).ToList(),
+                var clientRecords = new ClientsRecordView
+                {
+                    ClientName = (from c in _context.Customers
+                                  where c.CustomerId == id
+                                  select new ClientsRecord
+                                  {
+                                      Name = c.FirstName + " " + c.LastName,
+                                  }).ToList(),
+
+                    ClientOrders = (from o in _context.Orders
+                                    join oi in _context.OrderItems on o.OrderId equals oi.OrderId
+                                    join p in _context.Products on oi.ProductId equals p.ProductId
+                                    where o.CustomerId == id
+                                    select new ClientsRecord
+                                    {
+                                        OrderId = o.OrderId,
+                                        ProductName = p.Name,
+                                        Quantity = oi.Quantity,
+                                        UnitPrice = p.UnitPrice,
+                                    }).ToList(),
+                };
+                if (clientRecords.ClientOrders.Count > 0)
+                {
+                    _logger.Info("Returning orders found for this client");
+                }
+                else
+                {
+                    _logger.Info("No orders found for this client");
```

Comment:
What do you really care about logging wise here if this were a production application? If you opened up a logging system to view your logs, would this make sense? I think these would be more "Debug" logs so you know the outcome. But, not something you'd track in production.